src/components/Trade/hooks/useTradeRoutes/useTradeRoutes.ts

```diff
@@ -18,12 +19,16 @@ export const useTradeRoutes = (): {
   const buyAsset = getValues('buyAsset')
   const sellAsset = getValues('sellAsset')
   const assets = useSelector(selectAssets)
+  const asset = assetStore.getState()
```

Comment:
there's a much easier way to get the current asset id - take a look here https://github.com/shapeshift/web/blob/develop/src/pages/Assets/Asset.tsx#L57

Reply:
Thanks for reviewing, in fact, this was my first attempt, I called useAsset from useTradeRoutes at first, but it returned an error due to the fact that both Asset.tsx params { chain, tokenId } were coming undefined, so I thought of creating a redux store instead of refactoring an already functional and important hook
![image](https://user-images.githubusercontent.com/60587527/154089515-18042b59-32a6-4400-8d3b-5d19c16f5ff8.png)